inference-engine/src/readers/ir_reader/ie_ir_parser.hpp

```diff
@@ -256,6 +261,43 @@ class V10Parser : public IParser {
             stringToType<double>(val, value);
             adapter.set(value);
         }
+        void on_adapter(const std::string& name, ngraph::ValueAccessor<void*>& adapter) override  {
+            std::vector<std::string> value;
+            pugi::xml_node dn = node.child("data");
+            auto type = XMLParseUtils::GetStrAttr(node, "type");
+
+            if (dn.empty())
+                THROW_IE_EXCEPTION << "No attrtibutes defined for " << type << " op!";
+
+            if (getParameters<std::string>(dn, name, value)) {
+                auto data = const_cast<char*>(reinterpret_cast<const char*>(adapter.get_ptr()));
```

Comment:
Is quite unusual to `const_cast` pointer to put there some data. `const_cast` should be used more for legacy C functions.

I assume, because `adapter` is not const, this should be used here:
```
auto data = static_cast<char*>(adapter.get_ptr());
```

Reply:
corrected.